amgut/lib/data_access/ag_data_access.py

```diff
@@ -111,23 +74,25 @@ def authenticateWebAppUser(self, username, password):
         web_app_user table. If successful, a dict with user innformation is
         returned. If not, the function returns False.
         """
-        sql = """SELECT  cast(ag_login_id as varchar(100)) as ag_login_id,
-                  email, name, address, city,
-                  state, zip, country,kit_password
-                FROM ag_login
-            INNER JOIN ag_kit USING (ag_login_id)
-            WHERE supplied_kit_id = %s"""
-        row = self._sql.execute_fetchone(sql, [username])
-        if row:
-            results = dict(row)
-
-            if not bcrypt.verify(password, results['kit_password']):
+        with TRN:
+            sql = """SELECT  cast(ag_login_id as varchar(100)) as ag_login_id,
```

Comment:
Are any of these details used? It looks like the `auth_handler` only checks that the result evaluates `True` or `False`, and I can't see where things like email, name, etc are used.